DocumentedExamples/Meridional_heat_transport.ipynb

```diff
@@ -4,8 +4,9 @@
    "cell_type": "markdown",
```

Comment:
<p>I think it's timing the groupby, but I don't think it includes the sum/cumsum, no.</p>

Reply:
Good. But I think I'll remove it.
Unless we add for all methods a `.load()` at the end so that the actual computation is done then and the timing then makes sense. What do you reckon?